flask_ldap_login/__init__.py

```diff
@@ -172,6 +183,11 @@ def bind_search(self, username, password):
                 else:
                     log.debug("Username/password OK")
                     break
+        if not results and self._raise_errors:
+            msg = "No users found matching search criteria: {}".format(user_search)
+            if found_user:
+                msg = "Username/password mismatch"
+            raise ldap.INVALID_CREDENTIALS(msg)
```

Comment:
This should probably always log (at INFO or DEBUG level), and raise an exception when `raise_errors` is set.